doc-surrealql_versioned_docs/version-latest/statements/relate.mdx

```diff
@@ -685,4 +685,71 @@ SELECT * FROM likes;
 		out: person:one
 	}
 ]
+```
+
+## Multiple graph tables vs. fields
+
+Imagine a database that holds detailed information on the relations between NPCs in a game that are made to be as realistic as possible. Two of the characters have a rocky past but finally a happy conclusion in which they end up married. During this period, we might have tracked their relationship by adding and removing graph edges between the two of them as they move from a stage of being friends, to dating, to hating each other, to finally ending up married.
+
+```surql
+-- These three relations would end up deleted
+RELATE person:one->friends_with->person:two;
+RELATE person:one->dating->person:two;
+RELATE person:one->hates->person:two;
+-- Finally this would be the graph edge connecting the two
+RELATE person:one->married->person:two;
+```
+
+This works well to track the current state of the relationship, but creating a more general table such as `knows` along with a number of fields can be a better method to track the changing relationship over time. The following shows the relationship between the two `person` records, along with a third record called `person:three` who went to the same school and once dated `person:one`.
+
+```surql
+RELATE person:one->knows->person:two SET
+    has_been_friends = true,
+    has_dated = true,
+    has_hated = true,
+    married_to = true;
+
+RELATE person:one->knows->person:three SET
+    same_high_school = true,
+    has_dated = true;
+```
+
+With these fields in place, it is possible to use a `WHERE` clause to do refined searches on relationships of a certain type.
+
+```surql
+SELECT id, ->knows->person FROM person:one;
+SELECT id, ->knows[WHERE has_dated]->person FROM person:one;
+SELECT id, ->knows[WHERE same_high_school AND has_dated]->person FROM person:one;
+```
+
+Because the `WHERE` clause simply checks for [truthiness](/docs/surrealql/datamodel/values#values-and-truthiness) (whether a value is present and not empty), these fields do not necessarily need to be booleans and can even be complex objects.
+
+```surql
+RELATE person:one->knows->person:two SET
+	same_high_school = false,
```

Comment:
note: I detect indentation issues

Reply:
I don't see it. Looks alright in preview and on Surrealist